benchmarks/drop/requirements.txt

```diff
@@ -0,0 +1,2 @@
+inspect-ai==0.3.18
```

Comment:
Let's have this just be plain `inspect-ai` (so install the latest version). I am worried that pinning will miss important bug fixes and might lead people to believe they _must_ use an older version for the eval to work.

Reply:
If we aren't specifying the version, do we still mention `inspect-ai` in the requirements file or treat it as implied?

Reply:
I think we can treat it as implied